src/Common/src/Interop/Linux/libc/Interop.inotify.cs

```diff
@@ -32,7 +32,7 @@ internal static int inotify_rm_watch(int fd, int wd)
                 }
                 else
                 {
-                    System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
+                    global::System.Diagnostics.Debug.Fail("inotify_rm_watch failed with " + hr);
```

Comment:
Let's settle this offline and I'll loop back to the naming as a follow-up. I've left Interop.Libraries.SystemNative for now, but change Interop.System to Interop.Sys and got rid of the "global::" shenanigans.